spring-boot/admin/src/main/resources/proxy-docker-compose-template.yml

```diff
@@ -0,0 +1,14 @@
+services:
+
+  coatrack-proxy:
+    image: coatrack/coatrack-proxy:2.0.0-SNAPSHOT
```

Comment:
The version number will usually need to be aligned with the version of admin that is running. Maybe we should also make this configurable instead of hard-coding it here. To be further discussed.

Reply:
Done. I created a placeholder for the project-version in the template file. On runtime, Spring replaces it with the current project-version from the pom.xml. Is there still something to be discussed?